content/manuals/engine/release-notes/28.md

```diff
@@ -23,6 +23,27 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](/reference/api/engine/version-history.md).
 
+## 28.3.1
+
+{{< release-date date="2025-07-02" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 28.3.1 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A28.3.1)
+- [moby/moby, 28.3.1 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A28.3.1)
+
+### Bug fixes and enhancements
+
+- Fix `--use-api-socket` not working correctly when targeting a remote daemon. [docker/cli#6157](https://github.com/docker/cli/pull/6157)
```

Comment:
Looks like the CLI was built from the same commit as v28.3.0, so there's no changes in the CLI; https://github.com/moby/moby/compare/v28.3.0...v28.3.1